transcom/mymove
# INT B-19044 third times the charm

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19044)

## Summary

Prepare your butts cause this 19044 whiplash is gonna send us all out the window.
[First INT PR](https://github.com/transcom/mymove/pull/13783)
[Second INT PR](https://github.com/transcom/mymove/pull/13808)

I guess I forgot about adding the error count to the SC move details tab. So.. That's what this PR does.

### How to test

1. Create a PPM move as a customer and request an advance
2. Log in as a SC and find that move
3. You should now see an added count to the Move Details tab since the customer is requesting an advance
